### PR TITLE
update file header @link and @license URLs to match the new GitHub URL

### PR DESCRIPTION
**Pull request by tw2113**
Let me know if I need to revert the 3.x portion of the @license url

### CODE REVIEW

File: Slim/Handlers/NotFound.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ private function determineContentType($acceptHeader)
     {
         $list = explode(',', $acceptHeader);
         $known = ['application/json', 'application/xml', 'text/html'];
-
```
Comment (akrabat) [inline Slim/Handlers/NotFound.php:103]: Please remove this change.

File: Slim/Router.php [hunk 2]
```diff
@@ -163,7 +163,7 @@ public function dispatch(ServerRequestInterface $request)
 
         $dispatcher = new GroupCountBasedDispatcher($this->getData());
         $uri = '/' . ltrim($request->getUri()->getPath(), '/');
-        
+
```
Comment (akrabat) [inline Slim/Router.php]: Please remove this change

**Comment by akrabat** [comment]
Please rebase so that this PR merges cleanly

**Comment by tw2113** [comment]
Will do, thanks for the feedback

**Comment by tw2113** [comment]
Gonna restart this.

**Pull request closed without merging**